Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: docs
+init:
```

Comment:
@vaibhavmule Can you add the apt and brew install commands in here by detecting the OS? https://gist.github.com/sighingnow/deee806603ec9274fd47

Reply:
I can add that, have you seen this https://github.com/socialcopsdev/camelot/commit/835278ed3d03c33c58e41b99bfbc43642886a1e6 and seen how a build is failing if I add that to `init:` section: https://travis-ci.org/socialcopsdev/camelot/jobs/438470439

Reply:
Checking.